pom.xml

```diff
@@ -180,7 +180,7 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
-			<version>2.9.10.3</version>
+			<version>2.9.10.5</version>
```

Comment:
Why was this upgraded?

Reply:
I can revert it back. Did not make this change manually.